docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc

```diff
@@ -320,84 +320,70 @@ more information, see <<cast_expression,CAST Expression>>.
+.7+| Character String Data Type .3+| Fixed-length character | CHAR[ACTER]          | Fixed-length character data            | 1 to 32707 characters^2^
+| NCHAR                | Fixed-length character data in predefined national character set | 1 to 32707 bytes^3^ ^7^
+| NATIONAL CHAR[ACTER] | Fixed-length character data in predefined national character set | 1 to 32707 bytes^3^ ^7^
+.4+| Variable-length character | VARCHAR                      | Variable-length ASCII character string | 1 to 32703 characters^4^
+| CHAR[ACTER] VARYING          | Variable-length ASCII character string | 1 to 32703 characters^4^
+| NCHAR VARYING                | Variable-length ASCII character string | 1 to 32703 bytes^4^ ^7^
+| NATIONAL CHAR[ACTER] VARYING | Variable-length ASCII character string | 1 to 32703 characters^4^ ^7^
+.6+| Datetime Data Types .6+| Date-Time | | Point in time, using the Gregorian calendar and a 24 hour clock system. The five supported designations are listed below.
 | YEAR 0001-9999 +
 MONTH 1-12 +
 DAY 1-31 +
- +
 DAY constrained by MONTH and YEAR +
  +
 HOUR 0-23 +
 MINUTE 0-59 +
 SECOND 0-59 +
 FRACTION(n) 0-999999 +
- +
 in which n is the number of significant digits, from 1 to 6
-(default is 6; minimum is 1; maximum is 6). Actual database storage is
-incremental, as follows:
+(default is 6; minimum is 1; maximum is 6). +
  +
+Actual database storage is incremental, as follows:
 YEAR in 2 bytes +
 MONTH in 1 byte +
 DAY in 1 byte +
 HOUR in 1 byte +
 MINUTE in 1
 byte SECOND in 1 byte +
 FRACTION in 4 bytes +
-| | DATE                         | Date                                   | Format as YYYY-MM-DD; actual database storage size is 4 bytes
-| | TIME                         | Time of day, 24 hour clock, no time precision. Format as HH:MM:SS; actual database storage size is 3 bytes
-| | TIME (with time precision)   | Time of day, 24 hour clock, with time precision | Format as HH:MM:SS.FFFFFF; actual database storage size is 7 bytes
-| | TIMESTAMP                    | Point in time, no time precision | Format as YYYY-MM-DD HH:MM:SS; actual database storage size is 7 bytes
-| | TIMESTAMP (with time precision) Point in time, with time precision | Format as YYYY-MM-DD HH:MM:SS.FFFFFF; actual database storage size is 1 byte
-| Interval | INTERVAL | Duration of time; value is in the YEAR/MONTH range or the DAY/HOUR/MINUTE/YEAR/SECOND/FRACTION range
+| DATE                         | Date                                   | Format as YYYY-MM-DD; actual database storage size is 4 bytes
+| TIME                         | Time of day, 24 hour clock, no time precision. | Format as HH:MM:SS; actual database storage size is 3 bytes
+| TIME (with time precision)   | Time of day, 24 hour clock, with time precision | Format as HH:MM:SS.FFFFFF; actual database storage size is 7 bytes
+| TIMESTAMP                     | Point in time, no time precision | Format as YYYY-MM-DD HH:MM:SS; actual database storage size is 7 bytes

```

Comment:
A timestamp without a precision specified defaults to TIMESTAMP(6), with microsecond precision and a storage length of 11.